### PR TITLE
Chore/add first integration test chatbot

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
I had this integration test in my stash for a while. I'm just adding it to the repo so we have an starting point to continue to test as much as possible all edge cases in `/chatbot`.

#### How should this be reviewed?
- Run `npm run all-tests`

### CODE REVIEW

File: test/integration/routes/chatbot.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+// app
+const app = require('../../../app');
+
+test.after.always(() => {
+  mongoose.connection.db.dropDatabase();
+});
+
+test('Should fail if request does not include a valid api key', async () => {
+  await request(app)
```
Comment (sergii-tkachenko) [inline test/integration/routes/chatbot.test.js]: It's an async function. Maybe it'll be better without using then?

File: test/utils/stubs.js [hunk 2]
```diff
@@ -6,12 +6,18 @@
 const underscore = require('underscore');
 
 module.exports = {
+  getNorthstarAPIBaseUri: function getNorthstarAPIBaseUri() {
+    return process.env.DS_NORTHSTAR_API_BASEURI || 'https://northstar-thor.dosomething.org/v1';
```
Comment (sergii-tkachenko) [inline test/utils/stubs.js]: Might be better to default to localhost instead.

File: test/utils/factories/user.js [hunk 3]
```diff
@@ -8,10 +8,10 @@ const stubs = require('../../utils/stubs');
 
 const chance = new Chance();
 
-module.exports.getValidUser = function getValidUser() {
+module.exports.getUser = function getUser(phoneNumber) {
```
Comment (sergii-tkachenko) [inline test/utils/factories/user.js]: I didn't have time to implement it in blink, but I'd recommend looking into [`factory_girl`](https://github.com/aexmachina/factory-girl). It may be more convenient, as it allows both to generate fields from chance.js and override them on demand (as your're doing here with a phoneNumber).

File: test/integration/routes/chatbot.test.js (hunk 1 above)
Comment (rapala61) [reply test/integration/routes/chatbot.test.js]: You're right! I'm going to use the [.expect](https://www.npmjs.com/package/supertest#expectstatus-fn) function instead.

File: test/utils/stubs.js (hunk 2 above)
Comment (rapala61) [reply test/utils/stubs.js]: I've never user a local Northstar and the `.env.example` uses `https://northstar-fake.dosomething.org/v1` should I use that one?

File: test/utils/factories/user.js (hunk 3 above)
Comment (rapala61) [reply test/utils/factories/user.js]: I'll take a look in future integration tests.

**Comment by rapala61** [comment]
@sergii-tkachenko updated!

File: test/utils/stubs.js (hunk 2 above)
Comment (sergii-tkachenko) [reply test/utils/stubs.js]: Yep. The idea is you could use a lib that would fake responses from this domain. Can be really helpful with tests.

File: test/utils/factories/user.js (hunk 3 above)
Comment (sergii-tkachenko) [reply test/utils/factories/user.js]: Yep, just a note.

File: test/utils/stubs.js (hunk 2 above)
Comment (rapala61) [reply test/utils/stubs.js]: Ok. I use nock for tests. Updated.
